front/app/modules/commercial/content_builder/admin/components/RenderNode/index.tsx

```diff
@@ -52,14 +52,7 @@ export const getComponentNameMessage = (name: ComponentNamesType) => {
 
 const StyledBox = styled(Box)`
   ${({ isRoot }: { isRoot: boolean }) =>
-    isRoot
-      ? `cursor: auto;
-          padding: '4px',
-          width: '100%',
-          maxWidth: '1000px',
-          backgroundColor: '#fff',
-          minHeight: '160px',`
-      : `cursor:move;`}
+    isRoot ? `cursor: auto;` : `cursor:move;`}
 `;
 
```

Comment:
This fixes the UX issue Ben mentioned where the white background is gone from the content builder frame. I also removed the hardcoded width as asked.